smartcontract/contracts/RadaNFTToken.sol

```diff
@@ -33,29 +35,18 @@ contract RadaNFTToken is ERC721URIStorage, Ownable {
     function isWhitelister(address user) external view returns (bool) {
         return whitelist[user];
     }
-
-    function mint(string memory tokenURI) external onlyWhitelister {
+   
+    function mint(address _creator, string memory _tokenURI, uint _amount) external onlyWhitelister(_creator) {
+        
+        _mint(_creator, _tokenIds.current(), _amount, "");
```

Comment:
`       emit TransferSingle(operator, address(0), account, id, amount);`

Trong code cua ERC1155 no burn so tien nay. Co cach nao transfer no sang dia chi vi cua contract nay khong ae @khangt1k25 @januaryofmine

Reply:
No. Emit event is not burn any money.

Reply:
Cái này em và a Khánh đang nghiên cứu anh à, đang định tạo nhánh mới để thử xem chuyển thẳng vào ví của contract